packages/angular/cli/models/architect-command.ts

```diff
@@ -166,34 +166,32 @@ export abstract class ArchitectCommand extends Command<ArchitectCommandOptions>
       if (!targetSpec.project && this.target) {
         // This runs each target sequentially.
         // Running them in parallel would jumble the log messages.
-        return await from(this.getProjectNamesByTarget(this.target)).pipe(
+        return from(this.getProjectNamesByTarget(this.target)).pipe(
           concatMap(project => runSingleTarget({ ...targetSpec, project })),
           toArray(),
         ).toPromise().then(results => results.every(res => res === 0) ? 0 : 1);
       } else {
-        return await runSingleTarget(targetSpec).toPromise();
+        return runSingleTarget(targetSpec).toPromise();
       }
     } catch (e) {
       if (e instanceof schema.SchemaValidationException) {
         const newErrors: schema.SchemaValidatorError[] = [];
-        e.errors.forEach(schemaError => {
+        for (const schemaError of e.errors) {
           if (schemaError.keyword === 'additionalProperties') {
             const unknownProperty = schemaError.params.additionalProperty;
             if (unknownProperty in options) {
               const dashes = unknownProperty.length === 1 ? '-' : '--';
               this.logger.fatal(`Unknown option: '${dashes}${unknownProperty}'`);
-
-              return 1;
             }
           }
           newErrors.push(schemaError);
-        });
+        };
```

Comment:
Is that a semicolon at the end of a for loop?

Reply:
yes :).  it's corrected in the lint fix commit.